src/commands/login.js

```diff
@@ -48,6 +48,35 @@ const passwordPrompt = {
   }
 };
 
+function askCredentials(argv) {
+  let prompts = [];
+  let credentials = {
+    email: argv.email,
+    password: argv.password
+  };
+
+  if (credentials.email) {
+    console.log(`Logging in as ${credentials.email}.`);
+  } else {
+    prompts.push(emailPrompt);
+  }
+
+  if (!credentials.password) {
+    prompts.push(passwordPrompt);
+  }
+
+  if (prompts.length === 0) {
+    return Promise.resolve(credentials);
+  }
+
+  return inquirer.prompt(prompts).then((answers) => {
+    return {
+      ...answers,
+      ...credentials
```

Comment:
Merging the credentials this way the email and password will become empty if they are not provided by the argv.